common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -431,7 +432,15 @@ public long getLastSeenNotificationTime() {
         return prefs.getLong(PREFS_LAST_SEEN_NOTIFICATION_TIME, 0);
     }
 
-    public void setPrefsLastSeenNotificationTime(long lastSeenNotificationTime) {
+    public void setLastSeenNotificationTime(long lastSeenNotificationTime) {
         prefs.edit().putLong(PREFS_LAST_SEEN_NOTIFICATION_TIME, lastSeenNotificationTime).apply();
     }
+
+    public void setLocalProfilePictureUri(String picturePath) {
+        prefs.edit().putString(PREFS_KEY_LOCAL_PROFILE_PICTURE_URI, picturePath);
```

Comment:
`.apply()` needs to be called to save the image path to preferences.

Reply:
Fixed here: https://github.com/dashevo/dash-wallet/pull/528/commits/649f0091a5dc9d8caaf4046096c9d15993998ff3